tests/integration/backward_compatible/proxy/cp/count_down_latch_test.py

```diff
@@ -44,14 +45,23 @@ def test_await_latch_zero_timeout(self):
         self.assertFalse(latch.await_latch(0))
 
     def test_await_latch_with_timeout(self):
-        timeout = 0.1
+        timeout = 1
```

Comment:
Why we increased this? I mean, can you explain a little bit in detail why it will help? Also was this test failing in the past? Why and when it started to fail?

Reply:
It started failing once we started running tests on Windows. 

It will show that we are in fact waiting. 0.095 seconds might not show that we are waiting for timeout, so I made it 1 second to show that we are in fact waiting for the timeout. Note that, 0.95 for Windows also shows that we are waiting for quite long time